bedrock/firefox/templates/firefox/switch.html

```diff
@@ -16,61 +18,70 @@
 {% block content %}
 {% include 'firefox/includes/hub/sub-nav.html' %}
 <main>
-  <div class="outer-container">
-    <div class="content">
-      <div class="inner-container">
-        <header>
-          <h1>{{ _('Switch from Chrome to Firefox in just a few minutes') }}</h1>
-          <p class="tagline">{{ _('Switching to Firefox is fast, easy and risk-free. Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.') }}</p>
-        </header>
+  {% call hero(
+    title=_('Switch from Chrome to Firefox in just a few minutes'),
+    desc=_('Switching to Firefox is fast, easy and risk-free. Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.'),
+    class='mzp-has-image mzp-t-product-firefox t-switch',
+    include_cta=False,
+    heading_level=1
+  ) %}
+  {% endcall %}
 
-        <ol>
-          <li>{{ _('Select what to take from Chrome.') }}</li>
-          <li>{{ _('Let Firefox do the rest.') }}</li>
-          <li>{{ _('Enjoy the web faster, all set up for you.') }}</li>
-        </ol>
+  <section class="c-steps mzp-l-content">
+    <ol class="c-steps-list">
+      <li class="c-steps-item">{{ _('Select what to take from Chrome.') }}</li>
+      <li class="c-steps-item">{{ _('Let Firefox do the rest.') }}</li>
+      <li class="c-steps-item">{{ _('Enjoy the web faster, all set up for you.') }}</li>
+    </ol>
+    <div class="c-steps-cta">
+    {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+    </div>
+  </section>
 
-        {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+  <section class="c-share">
+    <div class="mzp-l-content">
 
-        <footer>
-          <p>{{ _('You can use Firefox and still have Chrome. Chrome won’t change on your machine one bit.') }} {{ _('Share with your friends how to switch to Firefox') }}</p>
+      {% if l10n_has_tag('switch_201908') %}
```

Comment:
Note to make sure we expose these new strings for translation, post launch.

Reply:
Merged. Here is a note to remind you about your note.

Reply:
Ooops, still catching up on emails. There's a string change to the string change. I'll submit a new PR.